frontend/src/components/PerDrawer.js

```diff
@@ -46,7 +46,7 @@ const useStyles = makeStyles(theme => ({
   },
   drawerPaper: {
     width: drawerWidth,
-    background: theme.palette.background.default,
+    /* background: "#F2FCFF", */
   },
   drawerHeader: {
```

Comment:
Is background gone now?  Should the commented out line be gone?

Reply:
This looks like a leftover style that just needs to be deleted..

Reply:
@hhummel @crsmyz I apologize, this PerDrawer file is no longer in use.  I need to just remove it.  Will do so now.

Reply:
ok removed the PerDrawer file, it is now in the MainLayout file.